rnn/enas_dag_node.cxx

```diff
@@ -145,6 +148,9 @@ double ENAS_DAG_Node::activation_derivative(double value, double input, int act_
     if(act_operator == 2) return swish_derivative(value,input);
     if(act_operator == 3) return leakyReLU_derivative(input);
     if(act_operator == 4) return identity_derivative();
+
+    Log::fatal("ERROR: invalid act_operator: %d\n", act_operator); 
+    exit(1);
```

Comment:
@travisdesell addressing potential undefined behavior here. Not sure how `exit(1)` is handled by MPI, assuming it propagates to processes? 
g++ returns 0 in my test. Perhaps finding an appropriate default return value is best.

Reply:
exit(1) should be fine here, it will terminate everything. this is a case we shouldn't ever hit anyways.